### PR TITLE
Accompany slides to 3K PBMC material

### PR DESCRIPTION
**Pull request by mtekman**
Incorporates fixes for the tutorial too

### CODE REVIEW

File: topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/slides.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+
+<video style="width:100%;height:auto" controls>
+    <source src="https://zenodo.org/record/3687100/files/live_cellxgene.mp4" />
+</video>
+
+
+---
+
+## Test
```
Comment (bgruening) [inline topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/slides.html]: is that needed?

Comment (mtekman) [reply topics/transcriptomics/tutorials/scrna-scanpy-pbmc3k/slides.html]: No, I just could not get the slides the build for a long time so was testing various ways.

**Comment by mtekman** [comment]
ready for review, thanks all

**Comment by bgruening** [comment]
@hrhotz do you have time for this one? Otherwise, I will give it a try tomorrow evening.

Thanks @mtekman

**Comment by hrhotz** [comment]
@bgruening 
This is too much new stuff - sorry - I won't have time for a thorough tomorrow, or on Friday.
I am off next week, so I am trying to finish a lot of other stuff within the next two days

**Comment by mtekman** [comment]
Thanks guys

**Comment by bgruening** [comment]
Thanks a lot! Looks great, nice video.

**Pull request merged**